x-pack/plugins/uptime/public/components/functional/snapshot_histogram.tsx

```diff
@@ -25,14 +31,14 @@ export const SnapshotHistogram = ({ histogram }: SnapshotHistogramProps) => {
         name={i18n.translate('xpack.uptime.snapshotHistogram.series.upLabel', {
           defaultMessage: 'Up',
         })}
-        color="green"
+        color={primaryColor}
```

Comment:
To be safe I always recommend pulling from the `EuiVisColor` series for anything chart related because they're targeted towards color blindess in charts. https://elastic.github.io/eui/#/guidelines/colors

The rest of this looks solid.

Reply:
@snide the `primaryColor` value in this case is either `euiDarkVars.euiColorVis1` or `euiLightVars.euiColorVis1` - is that not an appropriate value to use?
https://github.com/elastic/kibana/pull/29439/files#diff-abd19a861ef67d295beeea16e5a44b0bR120
https://github.com/elastic/kibana/pull/29439/files#diff-abd19a861ef67d295beeea16e5a44b0bR126